scripts/restore_secrets.py

```diff
@@ -0,0 +1,186 @@
+# Copyright 2022 Google LLC
```

Comment:
File is mainly a directly copy of the one from the C++ SDK repo.  Mainly changes have been made to the paths.